api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -451,6 +506,7 @@ class GetIndividualOfferResponseModel(BaseModel, AccessibilityComplianceMixin):
     withdrawalType: offers_models.WithdrawalTypeEnum | None
     status: OfferStatus
     isNonFreeOffer: bool | None
+    eventOpeningHours: list[GetEventOpeningHoursResponseModel]
```

Comment:
En fait, il ne doit il y avoir qu'un seul `eventOpeningHours` d'actif ou pas d'`eventOpeningHours` du tout. Donc il faudrait avoir :
```suggestion
    eventOpeningHours: GetEventOpeningHoursResponseModel | None
```